config/transaction-exclusion-api/transaction-exclusion-app-docker.config.toml

```diff
@@ -0,0 +1,30 @@
+data-queryable-window-since-rejected-timestamp="P7D"
+
+[database]
+schema="linea_transaction_exclusion"
+read_pool_size=10
+read_pipelining_limit=10
+transactional_pool_size=10
+[database.read]
+host="postgres"
+port="5432"
+username="postgres"
+password="postgres"
+[database.write]
+host="postgres"
+port="5432"
+username="postgres"
+password="postgres"
+[database.cleanup]
+polling-interval="PT60S"
```

Comment:
Note: when deploying to prod I recommend a polling time 10+min. So it's not cleaning all the time.